README.md

```diff
@@ -55,7 +56,7 @@ AWS provides all the building blocks. superwerker adds the wiring and `how to` s
     1. Protect superuser (`root`) access to your AWS accounts
     2. Scanning for best practise violations and active threats against your infrastructure
     3. Backups enabled for all database and file systems
-4. Billing best practices built-in: Automatic cost control and budget alarm
+4. Billing best practices built-in: Automatic cost control, budget alarms and configuration for enhanced usability (PDF invoices as well as sharing your tax information and credits across the organization)
```

Comment:
I'd emphasize that the tax inheritance means one single AWS bill aggregated across all accounts